### DISCUSSION TITLE
How to change cmake type from Debug to Release ，or reverse

### DISCUSSION THREAD

**Discussion by cdjzzzz** [LizardByte/community#485] [Sunshine] [answered]
last month i was building sunshine with release type, but recently i found that i build it with debug.
I have no idea where i change the config. could anyone help me?

**Comment by ReenigneArcher** [comment, marked as answer]
Kind of impossible to give you a clear answer without knowing exactly how you are building Sunshine, but...

https://stackoverflow.com/questions/19024259/how-to-change-the-build-type-to-release-mode-in-cmake

**Reply by cdjzzzz** [reply]
thank you, it works. i'm using vscode to build Sunshine project. I just add   into  settings.json
```
"cmake.configureArgs": [
        "-DCMAKE_BUILD_TYPE:STRING=RE"
    ],
```
